src/core_plugins/console/public/src/input.js

```diff
@@ -13,232 +13,238 @@ let es = require('./es');
 let history = require('./history');
 import uiModules from 'ui/modules';
 
-var $el = $('#editor');
-var input = new SenseEditor($el);
-uiModules.get('app/sense').setupResizeCheckerForRootEditors($el, input, output);
+let cached;
+export default function getInput($el, $actionsEl, $copyAsCurlEl, output) {
+  if (!$el) return cached;
```

Comment:
If I read this correctly, `getInput()` (without any arguments) is supposed to return the input, and `getInput($el...)` is suppose to initialize the input. Maybe we should just split this function in two? Looks like `initializeInput()` would only be called in one place anyway.